indexer/database/bridge.go

```diff
@@ -39,7 +42,7 @@ type DepositWithTransactionHash struct {
 }
 
 type Withdrawal struct {
-	GUID                 string `gorm:"primaryKey"`
+	GUID                 uuid.UUID `gorm:"primaryKey"`
```

Comment:
What do these label things actually do?  I used them myself in the cli part without even really fully understanding this language feature

Reply:
gorm will reflect on these tags and uses them to build the appropriate query string.

For example without the primaryKey tag, it wouldnt know how to properly insert or fetch rows and would tread the intended id as a regular column

Reply:
tbh that part is intuitive it's more like "wtf is this language feature" that's confusing me. Seeing here it's kinda an extra property Tag ont he thing though which makes it look less wierd/magical https://medium.com/golangspec/tags-in-golang-3e5db0b8ef3e